### PR TITLE
[Settings] Change handle dialog

### PR DESCRIPTION
**Pull request by mozzius**
# Based on #5777

https://github.com/user-attachments/assets/35284427-db0c-4732-a7ba-08bc40f05aa5

### CODE REVIEW

**Review by surfdude29 — COMMENTED** [review]
These are two very pedantic suggested tweaks, so please feel free to ignore them: I think these two strings would look slightly better using an en dash rather than a hyphen. (I think most [style](https://www.thepunctuationguide.com/em-dash.html) [guides](https://www.merriam-webster.com/grammar/em-dash-en-dash-how-to-use) would instead recommend an em dash in these cases, but I'm not sure that would fit Bluesky's style?)

**Comment by mozzius** [comment]
I do love an em dash

File: src/screens/Settings/components/ChangeHandleDialog.tsx [hunk 1]
```diff
@@ -0,0 +1,600 @@
...
+            <Trans>Change Handle</Trans>
+          </Dialog.HeaderText>
+        </Dialog.Header>
+      }
+      contentContainerStyle={[a.pt_0, a.px_0]}>
+      <View style={[a.flex_1, a.pt_lg, a.px_xl]}>
+        {serviceInfoError ? (
+          <ErrorScreen
+            title={_(msg`Oops!`)}
+            message={_(msg`There was an issue with fetching your service info`)}
```
Comment (surfdude29) [inline src/screens/Settings/components/ChangeHandleDialog.tsx:102]:
```suggestion
            message={_(msg`There was an issue fetching your service info`)}
```
slight suggested tweak

**Comment by haileyok** [comment]
Large font sizes make it quite hard to input a handle here, might need to adjust that.

Also note the "Cancel" button gets kinda messed up w/ the header.

Screenshot 2024-10-31 at 12 52 27 PM

**Comment by mozzius** [comment]
Fixed by setting a maxwidth. I'll fix the header separately

Simulator Screenshot - iPhone 16 - 2024-10-31 at 20 31 49

**Pull request merged**